modules/plugin/coverage-multidim/netcdf/src/main/java/org/geotools/imageio/netcdf/VariableAdapter.java

```diff
@@ -883,7 +884,7 @@ private void initRange() {
                 }
                 unit = SimpleUnitFormat.getInstance().parse(unitString);
                
-            } catch (IllegalArgumentException iae) {
+            } catch (ParserException parseException) {
```

Comment:
@aaime NetCDF failures came down to this one change, the exception changed. It is thrown when parsing unitsString "1" for example.